Google.Api.Generator/Formatting/WhitespaceFormatter.cs

```diff
@@ -151,6 +151,13 @@ public override SyntaxNode VisitArgumentList(ArgumentListSyntax node)
             return node;
         }
 
+        public override SyntaxNode VisitNameColon(NameColonSyntax node)
```

Comment:
Is this syntax used for inheritance as well? Just wondering whether it's going to cause us problems when we want a space before the colon for inheritance, but not for named arguments.

Reply:
No, inheritance uses `VisitBaseList`, which includes the `:` prefix of the base list.